vendor/golang.org/x/exp/trace/tracev1.go

```diff
@@ -4,24 +4,24 @@
 
 // Code generated by "gen.bash" from internal/trace; DO NOT EDIT.
 
-//go:build go1.21
+//go:build go1.23
 
-// This file implements conversion from old (Go 1.11–Go 1.21) traces to the Go
-// 1.22 format.
+// This file implements conversion from v1 (Go 1.11–Go 1.21) traces to the v2
```

Comment:
This is something I noticed last week; there's a v2 now, and v1 is implemented using v2; I wondered; should we be switching to v2?

But I'm now familiar at all with how it's used, so wanted to ask you 😅

Reply:
Oh! You are SO right! I completely missed the .. rather relevant .. `internal/` in the path!

I recall I was about to vendor an update, then noticed the `v1` / `v2`, and knowing that `golang.org/x/exp` is known for being experimental and breaking changes can happen, made a mental note "check if we should update our uses, because for this module we should keep up with changes made".

But nothing to do then, thanks!!